Pear/Profile/MatchProfileViewController.swift

```diff
@@ -0,0 +1,230 @@
+//
+//  MatchProfileViewController.swift
+//  Pear
+//
+//  Created by Lucy Xu on 4/28/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import GoogleSignIn
+import Kingfisher
+import UIKit
+
+class MatchProfileViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private var chatStatus: ChatStatus?
+    private let match: Match?
+    private var meetupStatusView: MeetupStatusView?
+    private var pair: User?
+    private var profileSections = [ProfileSectionType]()
+    private let profileTableView = UITableView(frame: .zero, style: .plain)
+    private let user: User?
+    private var reachOutButton = UIButton()
```

Comment:
Should this be alphabetized?  Everything else seems to be in alphabetical order besides reachOutButton